Submodule 3rdparty/tokenizers-cpp contains modified content
diff --git a/3rdparty/tokenizers-cpp/CMakeLists.txt b/3rdparty/tokenizers-cpp/CMakeLists.txt
index cfab093..a941d2d 100644
--- a/3rdparty/tokenizers-cpp/CMakeLists.txt
+++ b/3rdparty/tokenizers-cpp/CMakeLists.txt
@@ -8,7 +8,11 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 include(FetchContent)
 
 # update to contain more rust flags
-set(TOKENIZERS_CPP_RUST_FLAGS "")
+if(NODE)
+  set(TOKENIZERS_CPP_RUST_FLAGS -Ctarget-feature=+crt-static)
+else()
+  set(TOKENIZERS_CPP_RUST_FLAGS "")
+endif()
 set(TOKENIZERS_CPP_CARGO_TARGET "")
 
 # extra link libraries
